lib/tasks/db.rake

```diff
@@ -0,0 +1,37 @@
+require 'csv'
+require 'open-uri'
+
+namespace :db do
+  desc "Seed the database with degrees from a Degree Seeds CSV file"
+  task :seed_degrees, [:file_url] => :environment do |t, args|
+    csv_data = URI.open(args[:file_url])
+    CSV.new(csv_data, encoding: "bom|utf-8", headers: true).each.with_index(2) do |row, i|
+      Rails.logger.info("Processing row " + i.to_s)
+      degree = Degree.from_csv(row)
+      if degree.name_dspace.blank?
+        degree.update(name_dspace: row['Degree Name DSpace'])
+        Rails.logger.info("Degree DSpace name added: " + degree.name_dspace)
+      end
+      unless degree.degree_type
+        degree.update(degree_type: DegreeType.find_by(name: row['THING Degree Type']))
+        Rails.logger.info("Degree type added: " + degree.degree_type.name)
```

Comment:
I'm not sure what's up, but something in this logger statement is broken when I run this locally:

```
TypeError: no implicit conversion of nil into String
/Users/jprevost/projects/thing/lib/tasks/db.rake:17:in `+'
/Users/jprevost/projects/thing/lib/tasks/db.rake:17:in `block (3 levels) in <main>'
/Users/jprevost/projects/thing/lib/tasks/db.rake:8:in `with_index'
/Users/jprevost/projects/thing/lib/tasks/db.rake:8:in `block (2 levels) in <main>'
```

Reply:
That doesn't happen for me. Are your local degree types broken? :)

Reply:
Yup. I didn't do normal seeds before these specific seeds. I already have a separate docs ticket that captures the need to document how to seed for various scenarios so I'll resole this. Thanks!